cluster/manifests/stackset-controller/deployment.yaml

```diff
@@ -40,6 +40,9 @@ spec:
         - "--enable-configmap-support"
         - "--enable-secret-support"
         - "--enable-traffic-segments"
+{{- if eq .Cluster.ConfigItems.stackset_pcs_support_enabled "true" }}
+        - "--enable-pcs-support"
```

Comment:
Let's not do that yet because then we cannot flip the config-item in `stups-test` for running the e2e tests before merging the PR. (The current stackset-controller doesn't have this flag).

Let's just do the CRD changes for now, similar to https://github.com/zalando-incubator/kubernetes-on-aws/pull/7179/files

Reply:
That is necessary for the e2e.